xml/rmt_mirroring.xml

```diff
@@ -221,36 +259,76 @@ Jun 22 04:22:34 d31 systemd[1]: Started RMT Mirror timer.</screen>
    </para>
 <screen>&prompt.sudo;<command>rmt-cli repos list --all</command>
 +--------+-------------------------+-------------------------------------------+
-| SCC ID | Name                    | Description                               |
+| ID     | Name                    | Description                               |
 +--------+-------------------------+-------------------------------------------+
 [...]
 | 3061   | SUSE-PackageHub-15-Pool | SUSE-PackageHub-15-Pool for sle-15-x86_64 |
 [...]
 +--------+-------------------------+-------------------------------------------+
 
 &prompt.sudo;<command>rmt-cli repos enable 3061</command>
-Repository successfully enabled.
+Repository by ID 3061 successfully enabled.
 
 &prompt.sudo;<command>rmt-cli repos disable 3061</command>
-Repository successfully disabled.</screen>
+Repository by ID 3061 successfully disabled.
+
+To clean up downloaded files, please run 'rmt-cli repos clean'
+</screen>
 <tip>
  <title>Enabling and Disabling Multiple Repositories at Once</title>
  <para>
   To enable or disable multiple repositories at once, specify a space delimited
   list of their IDs, for example:
  </para>
-<screen>&prompt.sudo;<command>rmt-cli repos disable 2526 3263</command></screen>
+<screen>&prompt.sudo;<command>rmt-cli repos enable 2526 3263</command>
+Repository by ID 2526 successfully enabled.
+Repository by ID 3263 successfully enabled.
+
+&prompt.sudo;<command>rmt-cli repos disable 2526 3263</command>
+Repository by ID 2526 successfully disabled.
+Repository by ID 3263 successfully disabled.
+
+To clean up downloaded files, please run 'rmt-cli repos clean'
+</screen>
 </tip>
   </sect2>
  </sect1>
 
  <sect1 xml:id="sec-rmt-mirroring-delete">
   <title>Deleting Mirrored Data</title>
   <para>
-   After you disable mirroring of a repository or product as described in <xref linkend="sec-rmt-mirroring-enable-disable"/>, the mirrored data still remains on your local hard disk. This includes the mirrored RPM packages.
-   To remove the data, manually remove the corresponding directory:
+   After you disable the mirroring of a repository or product as described in <xref linkend="sec-rmt-mirroring-enable-disable"/>, the mirrored data still remains on your local hard disk. This includes the mirrored RPM packages.
   </para>
-  <screen>&prompt.sudo;<command>rm -r /usr/share/rmt/public/repo/SUSE/Products/<replaceable>PRODUCT</replaceable>/<replaceable>VERSION</replaceable>/<replaceable>ARCHITECTURE</replaceable>/</command></screen>
+  <para>
+   To delete a disabled repository's data, use the command <command>rmt-cli repos clean</command>. With this command, RMT
```

Comment:
```suggestion
   To delete disabled repository's data, use the command <command>rmt-cli repos clean</command>. With this command, RMT
```

Reply:
No, I would suggest reverting this change.